README.md

```diff
@@ -146,6 +146,12 @@ grpcurl -d @ grpc.server.com:443 my.custom.server.Service/Method <<EOM
 EOM
 ```
 
+Sometimes it might be necessary to read the payload from a file. This worked for some users:
```

Comment:
"This worked for some users" -- this wording feels really non-committal. We can make it authoritative.
```suggestion
To use a file as the request payload, redirect the file to `grpcurl`'s stdin:
```